docs/designers-developers/developers/block-api/block-registration.md

```diff
@@ -270,7 +270,7 @@ variations: [
 
 -   **Type:** `Object`
 
-Transforms provide rules for what a block can be transformed from and what it can be transformed to. A block can be transformed from another block, a shortcode, a regular expression, a file or a raw DOM node. Take a look at the [Block Transforms API](./block-transforms.md) for more info about each available transformation.
+Transforms provide rules for what a block can be transformed from and what it can be transformed to. A block can be transformed from another block, a shortcode, a regular expression, a file or a raw DOM node. Take a look at the [Block Transforms API](https://github.com/WordPress/gutenberg/blob/master/docs/designers-developers/developers/block-api/block-transforms.md) for more info about each available transformation.
```

Comment:
Thanks for fixing, the proper link is to not include the GitHub URL because docs are is used in multiple contexts, see [the Using Links section](https://developer.wordpress.org/block-editor/contributors/document/#using-links) of contributing docs.

```suggestion
Transforms provide rules for what a block can be transformed from and what it can be transformed to. A block can be transformed from another block, a shortcode, a regular expression, a file or a raw DOM node. Take a look at the [Block Transforms API](/docs/designers-developers/developers/block-api/block-transforms.md) for more info about each available transformation.
```